themes/custom_tmpl/GETTING_STARTED.md

```diff
@@ -0,0 +1,37 @@
+# Configuration
```

Comment:
A good start would be to include links to appropriate wiki pages. That's what I intend to do when we close this request: move the content to the wiki and provide a link.